src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -848,11 +832,161 @@ export class BrowsingContextImpl {
     }
   }
 
+  async #parseRect(clip?: BrowsingContext.ClipRectangle) {
+    if (!clip) {
+      return;
+    }
+    switch (clip.type) {
+      case 'viewport':
+        return {x: clip.x, y: clip.y, width: clip.width, height: clip.height};
+      case 'element': {
+        if (clip.scrollIntoView) {
+          throw new UnsupportedOperationException(
+            `'scrollIntoView' is currently not supported`
+          );
+        }
+        const sandbox = await this.getOrCreateSandbox(undefined);
```

Comment:
add a `// TODO:` here and link to https://github.com/GoogleChromeLabs/chromium-bidi/issues/1213

google3 style:

```
// TODO: #1213: Use custom sandbox specifically for Chromium BiDi
```